packages/wallet/src/KeyManagement/util/mapHardwareSigningData.ts

```diff
@@ -63,6 +96,15 @@ const sortTokensCanonically = (tokens: Token[]) => {
   });
 };
 
+const sortTrezorTokensCanonically = (tokens: CardanoToken[]) => {
```

Comment:
Seems to be exactly the same for both Ledger and Trezor. Would be good to generalize or sort before converting to device-specific types?